packages/testing-karma/README.md

```diff
@@ -84,12 +84,12 @@ By default karma runs all your test files. To test only a single file or folder,
 Grep supports file globs.
 
 ### Watch mode
-Use `npm run test -- --auto-watch=true --single-run=false` to keep karma running. Any code changes will trigger a re-run of your tests.
```

Comment:
that is actually correct as the first "--" is the separation between the npm command then the "real" command
e.g.
```bash
npm run test -- --auto-watch=true --single-run=false
# executes
karma run --auto-watch=true --single-run=false
```

Reply:
ahh okay I see. I was not able to run it with npm, probably because my npm client is set to yarn. So I ran it with yarn and it didn't need the "--" separation. Will add that back.